deployment/hasura/migrations/Aerie/18_plan_model_migration/up.sql

```diff
@@ -0,0 +1,405 @@
+alter table merlin.plan_snapshot
+add column model_id integer references merlin.mission_model on delete set null default null;
+
+-- Backfill the model_id column for existing snapshots with the current model_id for the snapshot's plan
+update merlin.plan_snapshot snap
+set model_id = plan.model_id
+from merlin.plan
+where plan.id = snap.plan_id;
+
+
+-- Alter snapshot functions
+drop function merlin.create_snapshot();
```

Comment:
Major: wrong function signature:
```suggestion
drop function merlin.create_snapshot(_plan_id integer);
```

Reply:
Good catch, fixed in ac9c10f7cc1b2300d1a6d8cb39fc85e9b8ca3705